src/mail/view/MailViewerToolbar.ts

```diff
@@ -164,6 +171,37 @@ export class MailViewerActions implements Component<MailViewerToolbarAttrs> {
 		}
 	}
 
+	private renderExportFailedEmailsDialog(mailList: Mail[]) {
+		const lines = mailList.map((mail) => ({
+			cells: [mail.sender.address, mail.subject],
+			actionButtonAttrs: null,
+		}))
+
+		const dialog = Dialog.createActionDialog({
+			title: lang.get("failedToExport_title"),
+			child: () =>
+				m("", [
+					m(".pt-m", lang.get("failedToExport_msg")),
+					m(ExpandableTable, {
+						title: () => lang.get("failedToExport_label", { "{0}": mailList.length }),
```

Comment:
This is purely subjective but I think just having the title be empty would look better because of the disparate fonts. ExpandableTable also has some weird padding that makes the "show" button just sit there randomly.
Ideally, the show button would be modified to say "show {count} failures" with proper alignment/spacing